docs/specification.md

```diff
@@ -403,8 +407,29 @@ This specification defines three standard transport protocols: [JSON-RPC Transpo
 
 The A2A protocol defines a canonical data model using Protocol Buffers. All transport implementations **MUST** provide functionally equivalent representations of these data structures.
 
+"Normative Source" Principle:
+
+The file `specification/grpc/a2a.proto` is the single authoritative normative definition of all protocol data objects and request/response messages. A generated JSON artifact (`specification/json/a2a.json`, produced at build time and not committed) MAY be published for convenience to tooling and the website, but it is a non-normative build artifact. SDK language bindings, schemas, and any other derived forms **MUST** be regenerated from the proto (directly or via code generation) rather than edited manually.
+
+Change Control and Deprecation Lifecycle:
+
+- Introduction: When a proto message or field is renamed, the new name is added while existing published names remain available for at least one subsequent minor release.
```

Comment:
Should this not say until the next major release?  I had assumed we would attempt to confirm to semantic versioning.